Packs/OpenAI/Integrations/OpenAI/README.md

```diff
@@ -1,63 +1,238 @@
+
+#### Command example
+```!openai-chatgpt prompt=`Give detailed analysis of the following using the following form:\n1- Analysis (Incident Description)\n2- Impact Analysis\n3- Action or Recommendations (Must be detailed)\n\nThere is a security incident with the following information:\n1- Title: ${incident.name}\n2- Indicator of compromise: ${ExtractedIndicators}\n3- Indicator of compromise reputation: ${Indicators.Score}\n...` model="gpt-35-turbo"```
+#### Context Example
+```json
+{
+    "OpenAI": {
+        "ChatGPTResponse": {
+            "ChatGPT Response": "1- Analysis:\nThe security incident with title XDR Incident # - 'Local Analysis Malware' generated by XDR Agent detected on host ABC involving user XYZ indicates that the XDR agent on host ABC has detected local analysis malware on the host, and it is associated with user XYZ.\n\n2- Impact Analysis:\nAn incident like this can be critical for an organization as local analysis malware can give attackers unauthorized access to the system and sensitive data, compromising the confidentiality and integrity of the organization's systems and data. The impact of this incident can result in a significant loss or damage to the system and can severely impact the organization's reputation.\n\n3- Action or Recommendations:\nIt is of utmost importance to immediately respond to such an incident and take action to contain the malware and prevent further damage. Below are some recommendations that can help in addressing this incident:\n\n- First, isolate the infected machine from the network to contain the malware's spread.\n- Check other systems for similar indicators of compromise.\n- Perform a thorough analysis to identify the malware's behavior and its impact on the system.\n- If the malware is a known strain, make sure to scan all other systems and implement the appropriate patches and updates to eliminate it.\n- If a new strain of malware, submit it for analysis to security vendors and take action based on their recommendations.\n- Reset the user's password and monitor their activities.\n- Look for any signs of data exfiltration or other suspicious activities.\n- Train employees on how to be more vigilant and proactively identify potential security threats.\n\nBy following these recommendations, organizations can significantly reduce the impact of such incidents and protect their systems and data from further damage. It is essential to have an incident response plan in place to handle such incidents promptly and effectively. Organizations should also make sure to perform regular vulnerability and incident assessments to identify and address any potential security risks.",
+            "Created Time": 1687968757,
+            "Model": "gpt-35-turbo",
+            "Number of Completion Tokens": 371,
+            "Number of Prompt Tokens": 101,
+            "Number of Total Tokens": 472,
+            "id": "chatcmpl-7WS4T5dI4r0RR8ZRa19QCM0cERlKt"
```

Comment:
Is this considered something private? Or can it be used publicly?

Reply:
Nope, it's not private, I don't see any issues with including this. The `id` is just a unique ID for each API response. Both [OpenAI](https://platform.openai.com/docs/api-reference/chat) and [Azure OpenAI](https://learn.microsoft.com/en-us/azure/ai-services/openai/reference) include examples like this in our docs. The number of tokens is just determined from the contents of the prompt/response so it is not sensitive either. 

Anyway I'll change it to a dummy ID `chatcmpl-123` like OpenAI does in their docs.